xbmc/interfaces/builtins/AddonBuiltins.cpp

```diff
@@ -43,6 +44,28 @@
 
 using namespace ADDON;
 
+/*! \brief Install an addon.
+ *  \param params The parameters.
+ *  \details params[0] = add-on id.
+ */
+static int InstallAddon(const std::vector<std::string>& params)
+{
+  if (params.size())
+  {
+    const std::string& addonid = params[0];
+
+    AddonPtr addon;
+    if (!CAddonMgr::GetInstance().GetAddon(addonid, addon))
+      CAddonInstaller::GetInstance().InstallModal(addonid, addon);
+    else
+      CLog::Log(LOGNOTICE, "InstallAddon called with an addonid of an installed addon.");
```

Comment:
This too seems unnecessary to me. Log errors and debug info. There's no need to log that something expected happened..